pkgs/development/ocaml-modules/rdbg/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, buildDunePackage, fetchurl, num, lutils, ounit}:
+
+buildDunePackage rec {
+  pname = "rdbg";
+  version = "1.196.12";
+
+  useDune2 = true;
```

```suggestion
  useDune2 = true;

  minimalOCamlVersion = "4.07";
```

The `rdbg` library won’t build with OCaml < 4.07.

---

I didn't know about `minimalOcamlVersion`, I applied this attribute to all `dune2` packages, for `ledit` I used the `meta.broken` attribute.